### ISSUE TITLE
Too easy to disable all rules accidentally

### ISSUE THREAD

**Issue by vsiao** [palantir/tslint#1269]
It's very easy to accidentally disable all lint rules by forgetting a `:` after `tslint:disable`. Can we also allow the following syntax?

#### TypeScript code being linted

```ts
/* tslint:disable max-line-length */
(code)
/* tslint:enable max-line-length */
```

#### Actual behavior

All lint rules are disabled for `(code)`.

#### Expected behavior

Only `max-line-length` is disabled for `(code)`.

**Comment by adidahiya** [comment]
Agreed wholeheartedly. I've also been bitten by this a few times.
